doc/template.html

```diff
@@ -57,7 +57,10 @@ <h1>Node.js __VERSION__ Documentation</h1>
 
       <div id="toc">
         <h2>Table of Contents</h2>
+<details>
+<summary>Table of contents</summary>
         __TOC__
+</details>
       </div>
```

Comment:
I think we can get rid of the `<h2>` element to avoid the repetition. Also, I think we should add a `open` attribute so the content is visible by default.

```suggestion
      <details id="toc" open>
        <summary>Table of Contents</summary>
        __TOC__
      </details>
```